### PR TITLE
for completeness that if the tarball from Alex's website is already i…

### PR DESCRIPTION
**Pull request by youyou3**
…n /tmp/cifar10_data/

user could download it before.

### CODE REVIEW

File: tensorflow/models/image/cifar10/cifar10.py [hunk 1]
```diff
@@ -392,3 +392,5 @@ def _progress(count, block_size, total_size):
     statinfo = os.stat(filepath)
     print('Successfully downloaded', filename, statinfo.st_size, 'bytes.')
     tarfile.open(filepath, 'r:gz').extractall(dest_directory)
```
Comment (caisq) [inline tensorflow/models/image/cifar10/cifar10.py]: Can you just move this statement out of the if scope, so that repetition can be avoided?

Comment (youyou3) [inline tensorflow/models/image/cifar10/cifar10.py]: Yes, it's more simple. While it seems need to create a new PL?

**Review by caisq — APPROVED** [review]

**Comment by caisq** [comment]
@tensorflow-jenkins , test this please.

**Comment by caisq** [comment]
@youyou3 Can you please squash your two commits into one?

**Comment by youyou3** [comment]
@caisq Could you show how to squash those two commits?
BTW, the checks are almost done now  " 2 pending and 8 successful checks"

**Comment by caisq** [comment]
@youyou3 Please see example about how to squash git commits at: http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

**Comment by caisq** [comment]
Ping @youyou3 regarding squashing. We'll test this PR again and merge it once the squashing is done.

**Comment by youyou3** [comment]
@caisq OK, I will squash it later in my free time. The network connection is too bad. Thanks

**Comment by youyou3** [comment]
@caisq The squashing is done. Thanks.

**Comment by caisq** [comment]
@youyou3, thank you!

@tensorflow-jenkins test this please.

**Comment by caisq** [comment]
Merged. Thanks, @youyou3

**Pull request merged**